R/render.r

```diff
@@ -422,7 +422,66 @@ data_deps <- function(pkg, depth) {
       )
     )
   )
-  deps <- bslib::bs_theme_dependencies(bs_theme)
+
+  # map secondary to component-active-bg
+  # unless a value was set by the user
+  if (is.null(pkg$meta[["template"]]$bslib$`component-active-bg`)) {
```

Comment:
Why can't we do this in sass?

Reply:
I might have missed something but if I only write

```r
bslib::bs_add_variables(
  bs_theme,
  "component-active-bg" = "$secondary !default",
  .where = "declarations"
)
```

then the variable keeps its inheritance of `$primary`.

https://sass-lang.com/documentation/variables#default-values